Lib/asyncio/proactor_events.py

```diff
@@ -426,6 +427,122 @@ def _pipe_closed(self, fut):
             self.close()
 
 
+class _ProactorDatagramTransport(_ProactorBasePipeTransport):
+
+    def __init__(self, loop, sock, protocol, address=None,
+                 waiter=None, extra=None):
+        super(_ProactorDatagramTransport, self).__init__(loop, sock, protocol,
```

Comment:
Why do you pass arguments to `super`?
`super().__init__(loop, sock, <whatever>)` should work in Python 3 AFAIK